crates/sui-framework/sources/governance/validator.move

```diff
@@ -272,11 +270,19 @@ module sui::validator {
     }
 
     /// Process pending delegations and withdraws, called at the end of the epoch.
-    public(friend) fun process_pending_delegations_and_withdraws(self: &mut Validator, new_epoch: u64, ctx: &mut TxContext) {
+    public(friend) fun process_pending_delegations(self: &mut Validator, new_epoch: u64, _ctx: &mut TxContext) {
+        staking_pool::process_pending_delegation(&mut self.delegation_staking_pool, new_epoch);
+        // TODO: consider bringing this assert back when we are more confident.
```

Comment:
No time like the present :). Now that we have safe advance epoch, I think we should be bold with assertions in this code.